kolibri/plugins/learn/assets/src/vue/explore-page/index.vue

```diff
@@ -33,6 +33,8 @@
       {{ topic.description }}
     </p>
 
+    <span class="visuallyhidden">You can navigate groups of content through headings.</span>
+
```

Comment:
This needs to be excluded when explore page is NOT displaying groups of content in folders.